.gitignore

```diff
@@ -10,3 +10,4 @@
 /public/system
 /tags
 /tmp/*
+*.rdb
```

Comment:
Yes, I believe so. According to the Redis documents it is a snapshot is produced when n number of records change in x time. https://redis.io/topics/persistence#snapshotting